### PR TITLE
Changed private browsing error msg and appearance #922

### PR DESCRIPTION
**Pull request by mradavi**
### Related issues
[Issue: Login page / Replace "﻿Browser not supported" more informative text #922
](https://gitlab.ci.csc.fi/sds-dev/sd-connect/swift-browser-ui/-/issues/922)

### Changes Made

- Changed "Browser not supported" to what's provided on Figma 
- Liisa provided a Finnish translation for it
- Changed the element from <c-login-card-content> to <c-alert> to best reflect the design on figma

### CODE REVIEW

**Comment by blankdots** [comment]
please don't link gitlab issues directly

**Comment by csc-felipe** [comment]
Nice first PR!

When there's spelling error check, it's usually because there are new words. You can add them to the dictionary file that lives in `.github/config/`. You could put the word anywhere and use a sorting command, e.g. `sort "${file}" | tr "[:upper:]" "[:lower:]" | sort -u -o "${file}"`.

**Review by csc-felipe — CHANGES_REQUESTED** [review]
sd-connect has a few different "main pages", and you should update the other ones that have the same message as well: `LoginPassword.vue` and `IndexPage.vue`.

To be honest, I'm not very comfortable with the new message, because it's inacurate. The error message may also be displayed in a browser that is not supported, and not only because it's Firefox's private mode.

My suggestion has been to add an entry to user documentation which states the supported browsers, and add a link to it from the error message. But I'm ok with other accurate solutions as well.

cc @ainoc

File: swift_browser_ui_frontend/src/common/lang.js [hunk 1]
```diff
@@ -46,10 +46,11 @@ let default_translations = {
         inUse: "Bucket name already in use.",
         invalidName: "Bucket name is invalid.",
         createFail: "Bucket creation failed.",
-        idb: "Browser not supported.",
+        idb: "Firefox in private mode is not supported.",
```
Comment (csc-felipe) [inline swift_browser_ui_frontend/src/common/lang.js:49]: It looks like you are not using this string anywhere

**Review by csc-felipe — CHANGES_REQUESTED** [review]
The text string is good now.

However, the template files are not all updated yet.

The changes have to be in all index pages that have it. 
You could search the repository for `message.error.idb` e.g. `grep -r message.error.idb`.
I get
- `LoginPassword.vue`
- `IndexPage.vue`
- `IndexOIDCPage.vue`

So it looks like one of them still need to be updated.

File: swift_browser_ui_frontend/src/common/lang.js (hunk 1 above)
Comment (mradavi) [reply swift_browser_ui_frontend/src/common/lang.js:49]: It's used in IndexPage as a card title

**Comment by csc-felipe** [comment]
When you are done with the review comments, you can [dismiss the review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/dismissing-a-pull-request-review) and [request a new one](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review).

For choosing who to ask for a review, you can check who has been reviewing and committing recently in the repository.

File: swift_browser_ui_frontend/src/common/lang.js (hunk 1 above)
Comment (csc-felipe) [reply swift_browser_ui_frontend/src/common/lang.js:49]: Why is it different in `IndexPage`?

**Comment by mradavi** [comment]
@csc-felipe I didn't update the design of IndexPage since it wasn't included in the issue. The text is updated. 

If the design needs changing there as well, then it would make sense to update the rest of the many error messages on that page as well. IMO warrants a separate issue so that on the design end it could be verified. Perhaps @ainoc would have some input on that?

Current view of IndexPage:
Screenshot 2023-02-13 at 13-15-30 SD Connect

**Comment by csc-felipe** [comment]
Fair enough, sorry about the noise

**Review by csc-felipe — APPROVED** [review]

**Pull request merged**